content/projekte/2016-11-28-cologne-openair.md

```diff
@@ -16,9 +16,6 @@ collaborators:
 - name: Martin Over
 - name: Reinhard Nickels
 
-tags:
```

Comment:
Hat es einen besonderen Grund, dass der Tag in diesem alten Beitrag entfernt wird?

Reply:
Nein... soweit ich verstanden habe werden die nicht mehr genutzt.

Reply:
Sofern kein technischer Grund besteht einen altes Projekt anzupassen, rate ich dazu keine Aenderungen vorzunehmen.
Bitte kehre zur urspruenglichen Version der Datei zurueck!